### PR TITLE
chore: fix for typo persistance to persistence (GH-7330)

### LINKED ISSUE #7330: Typo in filename state-persistance.service.spec.ts
persistance -> persistence

### PR DESCRIPTION
**Pull request by ravulapati-valtech**
fix for typo persistance to persistence

closes GH-7330

### CODE REVIEW

**Comment by Platonn** [comment]
Would you mind please:
1. change PR prefix from `fix: ` to `chore: `?
2. add in the footer of the PR's description: `closes GH-7330`

For more, see https://sap.github.io/spartacus-docs/commit-guidelines/

File: projects/core/src/state/services/state-persistance.service.spec.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ const winRef = {
   },
 } as WindowRef;
 
-describe('StatePersistanceService', () => {
```
Comment (Platonn) [inline projects/core/src/state/services/state-persistance.service.spec.ts:37]: Thanks for contribution. Can we also rename the files?

Comment (ravulapati-valtech) [reply projects/core/src/state/services/state-persistance.service.spec.ts:37]: @Platonn, thanks for the feedback, renamed file from state-persistance.service.spec.ts to state-persistence.service.spec.ts

**Comment by Platonn** [comment]
Our pipeline has a problem to allow for branches from outsite of spartacus repo, so I created a duplicate branch and PR with your commits preserved https://github.com/SAP/spartacus/pull/7405

**Pull request closed without merging**